fedn/fedn/network/combiner/aggregators/fedopt.py

```diff
@@ -34,16 +38,7 @@ def __init__(self, storage, server, modelservice, round_handler):
         self.v = None
```

Comment:
what are attributes v and m? No explanation of these. If they are not relevant to user name them _v and _m.

Reply:
Added a comment to explain what they are for.